CONTRIBUTING.md

```diff
@@ -28,6 +32,20 @@ As with all pieces of software, you may end up running into bugs. Please submit
 
 The best bug reports include a detailed way to predictably reproduce the issue, and possibly even a working example that demonstrates the issue.
 
+### Ongoing Participation
+
+We actively encourage ongoing participation by community members.
+
+* **Discussions** for implementations, designs, etc. often happen in the [amphtml-discuss Google Group](https://groups.google.com/forum/#!forum/amphtml-discuss) and the [amphtml Slack](https://docs.google.com/forms/d/1wAE8w3K5preZnBkRk-MD1QkX8FmlRDxd_vs4bFSeJlQ/viewform?fbzx=4406980310789882877).
+* **Weekly status updates** from individual community members are posted as GitHub issues labeled [Meeting Notes](https://github.com/ampproject/amphtml/issues?q=label%3A%22Meeting+Notes%22).  If you have a weekly status update related to your work on AMP that you'd like to share with the community please add it as a comment on the relevant Meeting Notes issue.
+* **Weekly design reviews** are held as video conferences via Google Hangouts on Wednesdays at [1pm Pacific](https://www.google.com/?#q=1pm+pacific+in+local+time).  Design reviews are used to discuss/refine engineering designs after an initial draft of the design has been created and shared with the community.
+  * We use GitHub issues labeled [Type: Design Review](https://github.com/ampproject/amphtml/issues?q=label%3A%22Type%3A+Design+Review%22) to track design reviews.  The Design Review issue for a given week will have a link to the design docs being discussed that week as well as a link to the Hangout.
+  * When you attend a design review please read through the design docs before the review starts.
+  * If you have an engineering design you would like to discuss at a design review:
+    * Document your design in a shared Google Document open to public comments. Take a look at [Design docs - A design doc](https://medium.com/@cramforce/design-docs-a-design-doc-a152f4484c6b) for tips on putting together a good design doc.
```

Comment:
I've added a comment about short docs being okay and filed Issue #7281 for tracking adding some example design docs when we have some.

(I also added a line to the pre-review section indicating it's fine to request a pre-review before the doc is complete.)